src/tarball-file-system.ts

```diff
@@ -48,4 +48,15 @@ export class TarballFileSystem implements FileSystem {
     const {files} = await this.#getUnpackResult();
     return files.reduce((acc, file) => acc + file.data.byteLength, 0);
   }
+
+  async hasTypeScriptConfig(): Promise<boolean> {
```

Comment:
i meant then we can drop this method since we can just call `fileExists('/tsconfig.json')`

keeps the interface a little more generic then, less tied into the implementation

Reply:
sorry i missunderstood this, gonna fix it asap 🫡